data/base-spells.yaml

```diff
@@ -632,111 +661,134 @@ spell_data:
     prep: prepare
     starlight_threshold: 0
     mana_type: lunar
+    recast: 1
   Arc Light:
     skill: Debilitation
     harmless: true
     abbrev: AL
     mana: 1
     mana_type: elemental
+    prep: prepare
+    recast: 1
   Artificer's Eye:
     skill: Augmentation
     abbrev: ART
     mana: 5
     recast: 1
     mana_type: lunar
+    prep: prepare
   Aspects of the All-God:
-    skill: Augmentation  # Also utility
+    skill: Augmentation # Also Utility
```

Comment:
I keep forgetting about these dual-skill spells. We ought to make this a list at some point...

Reply:
Someday, but not this day 😉